migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/MisoServiceManager.java

```diff
@@ -710,38 +700,28 @@ public void setDefaultPoolDao() {
 
   private void updatePoolDaoDependencies() {
     if (libraryDao != null) libraryDao.setPoolDAO(poolDao);
-    if (experimentDao != null) experimentDao.setPoolDAO(poolDao);
     if (partitionDao != null) partitionDao.setPoolDAO(poolDao);
     if (boxDao != null) boxDao.setPoolDAO(poolDao);
   }
 
-  public SQLExperimentDAO getExperimentDao() {
+  public HibernateExperimentDao getExperimentDao() {
```

Comment:
Sure. So long as they have a consistent name we can use for the Prometheus monitoring to grab during autowiring.